boards/arm/actinius_icarus/actinius_icarus_common.dts

```diff
@@ -82,6 +82,12 @@
 		output-ohms = <10000000>;
 		full-ohms = <(10000000 + 4700000)>;
 	};
+
+	sim_select: sim-select {
+		compatible = "actinius,sim-select";
```

Comment:
Please read this and fix the compatible: https://docs.zephyrproject.org/latest/guides/dts/bindings.html#vendor-prefixes

Reply:
thanks for the insight, removed `actinius,` prefix from the compatible.